### PR TITLE
Raspberry helloworld

### PR DESCRIPTION
**Pull request by BeckmaR**

### CODE REVIEW

File: org.yakindu.examples.raspberry.helloworld/.project [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>org.yakindu.examples.raspberry.helloworld</name>
+	<comment></comment>
+	<projects>
+		<project>coffee_machine_pro</project>
```
Comment (andreasmuelder) [inline org.yakindu.examples.raspberry.helloworld/.project]: should be raspberry hello world

File: org.yakindu.examples.raspberry.helloworld/index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    <p>This example shows how to use a statechart and some C code to
+    control a RGB LED attached to the Raspberry Pi.</p>
+    
+    <p>To compile it for the Pi, you need a cross compiler set up, or you need
+    to compile it directly on the Pi.</p>
+    
+    <p>The project was designed on a Linux machine and uses the pthread library.</p>
+    
+    <p>For further instructions, please see the associated <a href="">blog post</a>.</p>
```
Comment (andreasmuelder) [inline org.yakindu.examples.raspberry.helloworld/index.html]: Add Link

File: org.yakindu.examples.raspberry.helloworld/index.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  </div>
+
+  <div>
+    <h2>Images</h2>
```
Comment (andreasmuelder) [inline org.yakindu.examples.raspberry.helloworld/index.html]: Remove example section

File: org.yakindu.examples.raspberry.helloworld/metadata.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{
+	"id": "org.yakindu.sct.examples.raspberry.helloworld",
+	"title": "Raspberry Pi Hello World",
+	"description": "This example shows how to use a statechart and some C code to control a RGB LED attached to the Raspberry Pi.",
+	"previewImage": "header.jpg",
+    "author": "itemis AG",
+    "organization": "statecharts.org",
+    "license": "Creative Commons Attribution 4.0 International (CC BY 4.0)",
```
Comment (andreasmuelder) [inline org.yakindu.examples.raspberry.helloworld/metadata.json]: Why not EPL?

Comment (BeckmaR) [inline org.yakindu.examples.raspberry.helloworld/metadata.json]: I don't know, I believe it was in the example template.

File: org.yakindu.examples.raspberry.helloworld/.project (hunk 1 above)
Comment (BeckmaR) [inline org.yakindu.examples.raspberry.helloworld/.project]: Line number 6 was a reference to the project coffee_machine_pro which I just removed. Or did you mean the name tag?

File: org.yakindu.examples.raspberry.helloworld/index.html [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    
+    <p>The project was designed on a Linux machine and uses the pthread library.</p>
+    
+    <p>For further instructions, please see the associated <a href="https://blogs.itemis.com/en/how-to-program-your-raspberry-pi-with-statechart-tools-in-5-minutes">blog post</a>.</p>
+  </div>
+
+  <div>
+    <h2>Images</h2>
+
+    <p>This section demonstrates how to include an image.</p>
```
Comment (andreasmuelder) [inline org.yakindu.examples.raspberry.helloworld/index.html]: remove this

**Pull request merged**